TRIAGING.md

```diff
@@ -23,9 +23,11 @@ If you have an issue you'd like to bring forth please consider getting a link to
 
 1. Announcements: If there are any announcements to be made they will happen at the start of the meeting.
-2. Review of new issues: The meetings always start with reviewing all untriaged issues for the [security](https://github.com/opensearch-project/security/issues?q=is%3Aissue+is%3Aopen+label%3Auntriaged) repo and [security-dashboards](https://github.com/opensearch-project/security-dashboards-plugin/issues?q=is%3Aissue+is%3Aopen+-label%3Atriaged) repo. We will also provide an opportunity for any issues introduced since the last meeting but already triaged to be discussed.
-3. Backlog discussion: Next, we open the floor in case anyone from the audience wants to highlight an issue from the backlogs ([security](https://github.com/opensearch-project/security/issues?q=is%3Aissue+is%3Aopen+label%3Atriaged+-label%3A%22sprint+backlog%22) and [security-dashboards](https://github.com/opensearch-project/security-dashboards-plugin/issues?q=is%3Aopen+is%3Aissue+label%3Atriaged+-label%3A%22sprint+backlog%22+)).
-4. Sprint backlog review: Finally, we will review persisting issues in the sprint backlog for the [security](https://github.com/opensearch-project/security/issues?q=is%3Aissue+is%3Aopen+label%3A%22sprint+backlog%22) repo and [security-dashboards](https://github.com/opensearch-project/security-dashboards-plugin/issues?q=is%3Aopen+is%3Aissue+label%3A%22sprint+backlog%22) repo, checking to see that issues have not gone stale or been addressed.
+2. Review of new issues: The meetings always start with reviewing all untriaged [issues](https://github.com/search?q=label%3Auntriaged+is%3Aopen++repo%3Aopensearch-project%2Fsecurity+repo%3Aopensearch-project%2Fsecurity-dashboards-plugin&type=issues&ref=advsearch&s=created&o=desc) for the security and security-dashboards repositories.
+3. Review items that are not triaged, We will review any [issues](https://github.com/search?q=-label%3Atriaged+is%3Aopen++repo%3Aopensearch-project%2Fsecurity+repo%3Aopensearch-project%2Fsecurity-dashboards-plugin&type=issues&ref=advsearch&s=created&o=desc) that might have had the 'untriaged' label removed but require additional triage discussion.
+4. Open discussion: Next, we open the floor in case anyone from the audience wants to highlight an issue.
+5. Backlog discussion: Then, we review issues from the [backlogs](https://github.com/search?q=label%3A%22sprint+backlog%22+is%3Aopen++repo%3Aopensearch-project%2Fsecurity+repo%3Aopensearch-project%2Fsecurity-dashboards-plugin&type=issues&ref=advsearch&s=created&o=desc) security and security-dashboards repositories.
+6. Least recent discussed issue: Finally, to close out the meeting we will [review the oldest](https://github.com/search?q=+is%3Aopen++repo%3Aopensearch-project%2Fsecurity+repo%3Aopensearch-project%2Fsecurity-dashboards-plugin&type=issues&ref=advsearch&s=updated&o=asc) issues from either repository security and security-dashboards, to help identity issues that have languished.
```

Comment:
Do you want to move this up in the priorities? I know you had wanted to split the time 50/50 between general-security and Identity specific work but this looks like the first mention of Identity. I don't want you to feel short-shrifted.

Reply:
I was thinking about it, that fourth bullet, Open Discussion, is a great place to pull in ANY issues, which I think could also be cited from other OpenSearch-Project repos, or even fully external repos.  For now, I don't have a grand plan for reviewing the Identity project plan, and I don't want to forcefully inject a program review inside the security triage meeting.  If there are useful topics, I think they will organically be linked out from security issues, or members of the maintainers/identity project can always ask to review the topic.

It might make sense to build on this meeting or spin up a separate meeting, but I don't have a settled opinion at this time.  If anyone does, I'm open to suggestions 😄

Reply:
As long as you are happy with the ordering that seems fine to me. I just didn't want you to get Identity added to the agenda but then see it be an afterthought. We could always change things around later if you feel you need more specific feedback weekly.

Reply:
I would prefer have `Open discussion` as a higher priority item than this, as someone can bring a pressing issue (which most likely will have higher precedence over `Least recently discussed` issue. Also, there is a decent probability that there won't be more than 1 or 2 issue brought up by community, so we will be able to spare some time on backlog items and least discussed issues.
Having said that, if at any point in future we start getting larger audience that brings up more issues, maybe we can revisit the priorities or have a separate meeting for community issues.

Reply:
nit: `issues from both repositories, security and security-dashboards, to help identify issues that have languished.`